src/wp-includes/class-wp-image-editor.php

```diff
@@ -431,8 +431,10 @@ protected function get_output_format( $filename = null, $mime_type = null ) {
 	 */
 	public function generate_filename( $suffix = null, $dest_path = null, $extension = null ) {
 		// $suffix will be appended to the destination filename, just before the extension.
```

Comment:
Also maybe update the inline comment.

```suggestion
		// If not empty the $suffix will be appended to the destination filename, just before the extension.
```